_posts/2022-05-05-8.0.0-release.md

```diff
@@ -0,0 +1,230 @@
+
+#### CI
+
+* staticcheck is now run as part of CI to lint the Go code ([ARROW-15296](https://github.com/apache/arrow/pull/12540))
+* Travis builds on Arm64 for Go are no longer allowed to fail ([ARROW-15400](https://github.com/apache/arrow/pull/12214))
+
+## Java notes
+* Java 17 is now supported as a target and has been added to tested platforms
+* When scanning datasets an `ArrowReader` is now returned, which makes easier to create `VectorSchemaRoot` from it.
+* Java Documentation had a overall improvement, with few sections added and most sections rewritten as more clear tutorials
+* Overall improvements to FlightSQL support in Java
+* [Java Cookbook](https://arrow.apache.org/cookbook/java/index.html) is now available
+## JavaScript notes
+
+## Python notes
+
+In general, the Python bindings benefit from improvements in the C++ library
+(e.g. new compute functions); see the C++ notes above for additional details.
+In addition:
+
+* Tables and Datasets now support the ``join`` operation to perform `left`, `right`, `full` joins of `inner` or `outer` types. The result of the join operation will be a new table ([ARROW-14293](https://issues.apache.org/jira/browse/ARROW-14293)). See https://arrow.apache.org/docs/dev/python/compute.html#table-and-dataset-joins for examples.
+* Additional legacy keywords and properties of the ``ParquetDataset`` class have been deprecated and will issue a warning, in favor of functionality based on the `pyarrow.dataset` functionality ([ARROW-16119](https://issues.apache.org/jira/browse/ARROW-16119)).
+* It is now possible to create references to nested fields in a Table or Dataset using ``py.field("a", "b")`` ([ARROW-11259](https://issues.apache.org/jira/browse/ARROW-11259)).
+* Docstrings in ``Schema``, ``ChunkedArray``, ``Tensor``, ``RecordBatch``, ``parquet`` and ``Table`` now include examples, in the documentation, on how to use the methods and classes ([ARROW-15367](https://issues.apache.org/jira/browse/ARROW-15367)).
+* Reading and writing Parquet files now supports encryption ([ARROW-9947](https://issues.apache.org/jira/browse/ARROW-9947)). See the [docs](https://arrow.apache.org/docs/python/parquet.html#parquet-modular-encryption-columnar-encryption) for more details.
+* Support for `zoneinfo` (Python 3.9+) and `dateutil` timezones in conversion to Arrow data structures ([ARROW-5248](https://issues.apache.org/jira/browse/)).
+* Multiple bugfixes and segfaults resolved.
+
+## R notes
+
+This release includes:
+
+ * Support for over 20 additional `lubridate` and `base` date and time functions in Arrow dpylr queries,
+ * An API to allow external packages to define custom extension array types and custom conversions into Arrow types,
+ * Support for concatenating Arrays, RecordBatches, and Tables, including with `c()`, `rbind()` and `cbind()`.
+
+For more on what’s in the 8.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
+
+### Ruby
+
+### C GLib
+
```

Comment:
```suggestion
### C GLib

* Add `gaflight_client_close` ([[ARROW-15487](https://issues.apache.org/jira/browse/ARROW-15487)](https://issues.apache.org/jira/browse/ARROW-15487))
* Add `GParquetFileMetadata` and `gparquet_arrow_file_reader_get_metadata` ([[ARROW-16214](https://issues.apache.org/jira/browse/ARROW-16214)](https://issues.apache.org/jira/browse/ARROW-16214))
* Fix `GArrowGIOInputStream` so that all the data is completely read ([[ARROW-15626](https://issues.apache.org/jira/browse/ARROW-15626)](https://issues.apache.org/jira/browse/ARROW-15626))
* Add `garrow_string_array_builder_append_string_len` and `garrow_large_string_array_builder_append_string_len` ([[ARROW-15629)](https://issues.apache.org/jira/browse/ARROW-15629)](https://issues.apache.org/jira/browse/ARROW-15629)
* Add `GParquetRowGroupMetadata` ([[ARROW-16245](https://issues.apache.org/jira/browse/ARROW-16245)](https://issues.apache.org/jira/browse/ARROW-16245))
* Add `GParquetColumnChunkMetadata` ([[ARROW-16250](https://issues.apache.org/jira/browse/ARROW-16250)](https://issues.apache.org/jira/browse/ARROW-16250))
* Add `GArrowGCSFileSystem` ([[ARROW-16247](https://issues.apache.org/jira/browse/ARROW-16247)](https://issues.apache.org/jira/browse/ARROW-16247))
* Add `GParquetStatistics` and its family ([[ARROW-16251](https://issues.apache.org/jira/browse/ARROW-16251)](https://issues.apache.org/jira/browse/ARROW-16251))
* Add missing casts for `GArrowRoundMode` ([[ARROW-16296](https://issues.apache.org/jira/browse/ARROW-16296)](https://issues.apache.org/jira/browse/ARROW-16296))

```